central/declarativeconfig/manager_impl.go

```diff
@@ -0,0 +1,41 @@
+package declarativeconfig
+
+import (
+	"context"
+	"time"
+
+	"github.com/stackrox/rox/pkg/k8scfgwatch"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+const (
+	watchInterval        = 5 * time.Second
+	declarativeConfigDir = "/run/stackrox.io/declarative-configuration"
+)
+
+var (
+	log = logging.LoggerForModule()
```

Comment:
`log` is unused

Reply:
I've used it now within the `OnWatchError`